### PR TITLE
Updated copyright notice.

### PR DESCRIPTION
**Pull request by andrey-zelenkov**

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+   Copyright 2017-2024 Andrei Zeliankou
+   Copyright 2018-2024 Konstantin Pavlov
+   Copyright 2021-2024 Zhidao Hong
+   Copyright 2022-2024 Andrew Clayton
+   Copyright 2022-2024 Liam Crilly
+   Copyright 2023-2024 Dan Callahan
+   Copyright 2023-2024 Danielle De Leo
+   Copyright 2023-2024 Dylan Arbour
+   Copyright 2023-2024 Gabor Javorszky
+   Copyright 2023-2024 Ippolitov Igor
```
Comment (callahad) [inline NOTICE:14]: Should `Ippolitov Igor` be `Igor Ippolitov` here and elsewhere?

Also, while we're at it, can we change `Nginx, Inc.` to `NGINX, Inc.` in `src/test/nxt_rbtree1.c`, `src/test/nxt_rbtree1.h`, and `auto/endian`

Comment (andrey-zelenkov) [reply NOTICE]: Thanks, fixed!

**Comment by andrey-zelenkov** [comment]
```
 % git range-diff a4326214...11b90144
 
 1:  a4326214 ! 1:  11b90144 Updated copyright notice.
    @@ NOTICE
     +   Copyright 2023-2024 Danielle De Leo
     +   Copyright 2023-2024 Dylan Arbour
     +   Copyright 2023-2024 Gabor Javorszky
    -+   Copyright 2023-2024 Ippolitov Igor
    ++   Copyright 2023-2024 Igor Ippolitov
     +   Copyright 2023-2024 Taryn Musgrave
         Copyright 2021-2023 Alejandro Colomar
     -   Copyright 2022-2023 Andrew Clayton
    @@ NOTICE
         Copyright 2017-2022 Max Romanov
         Copyright 2021-2022 Oisín Canty
     
    + ## auto/endian ##
    +@@
    + # Copyright (C) Igor Sysoev
    + # Copyright (C) Andrew Clayton
    +-# Copyright (C) Nginx, Inc.
    ++# Copyright (C) NGINX, Inc.
    + 
    + 
    + nxt_feature="endianness"
    +
      ## docs/man/man8/unitd.8.in ##
     @@
     -.\" (C) 2017-2023, NGINX, Inc.
    @@ docs/man/man8/unitd.8.in
     +.\" (C) 2023-2024 Danielle De Leo
     +.\" (C) 2023-2024 Dylan Arbour
     +.\" (C) 2023-2024 Gabor Javorszky
    -+.\" (C) 2023-2024 Ippolitov Igor
    ++.\" (C) 2023-2024 Igor Ippolitov
     +.\" (C) 2023-2024 Taryn Musgrave
      .\" (C) 2021-2023 Alejandro Colomar
     -.\" (C) 2022-2023 Andrew Clayton
    @@ docs/man/man8/unitd.8.in: The socket address of Unit's control API.
     +(C) 2023-2024 Danielle De Leo
     +(C) 2023-2024 Dylan Arbour
     +(C) 2023-2024 Gabor Javorszky
    -+(C) 2023-2024 Ippolitov Igor
    ++(C) 2023-2024 Igor Ippolitov
     +(C) 2023-2024 Taryn Musgrave
      (C) 2021-2023 Alejandro Colomar
     -(C) 2022-2023 Andrew Clayton
    @@ pkg/deb/debian.module/copyright.unit-jsc11
     +   Copyright 2023-2024 Danielle De Leo
     +   Copyright 2023-2024 Dylan Arbour
     +   Copyright 2023-2024 Gabor Javorszky
    -+   Copyright 2023-2024 Ippolitov Igor
    ++   Copyright 2023-2024 Igor Ippolitov
     +   Copyright 2023-2024 Taryn Musgrave
         Copyright 2021-2023 Alejandro Colomar
     -   Copyright 2022-2023 Andrew Clayton
    @@ pkg/deb/debian.module/copyright.unit-jsc8
     +   Copyright 2023-2024 Danielle De Leo
     +   Copyright 2023-2024 Dylan Arbour
     +   Copyright 2023-2024 Gabor Javorszky
    -+   Copyright 2023-2024 Ippolitov Igor
    ++   Copyright 2023-2024 Igor Ippolitov
     +   Copyright 2023-2024 Taryn Musgrave
         Copyright 2021-2023 Alejandro Colomar
     -   Copyright 2022-2023 Andrew Clayton
    @@ pkg/deb/debian/copyright
     +   Copyright 2023-2024 Danielle De Leo
     +   Copyright 2023-2024 Dylan Arbour
     +   Copyright 2023-2024 Gabor Javorszky
    -+   Copyright 2023-2024 Ippolitov Igor
    ++   Copyright 2023-2024 Igor Ippolitov
     +   Copyright 2023-2024 Taryn Musgrave
         Copyright 2021-2023 Alejandro Colomar
     -   Copyright 2022-2023 Andrew Clayton
    @@ pkg/rpm/rpmbuild/SOURCES/COPYRIGHT.unit-jsc11
     +   Copyright 2023-2024 Danielle De Leo
     +   Copyright 2023-2024 Dylan Arbour
     +   Copyright 2023-2024 Gabor Javorszky
    -+   Copyright 2023-2024 Ippolitov Igor
    ++   Copyright 2023-2024 Igor Ippolitov
     +   Copyright 2023-2024 Taryn Musgrave
         Copyright 2021-2023 Alejandro Colomar
     -   Copyright 2022-2023 Andrew Clayton
    @@ pkg/rpm/rpmbuild/SOURCES/COPYRIGHT.unit-jsc8
     +   Copyright 2023-2024 Danielle De Leo
     +   Copyright 2023-2024 Dylan Arbour
     +   Copyright 2023-2024 Gabor Javorszky
    -+   Copyright 2023-2024 Ippolitov Igor
    ++   Copyright 2023-2024 Igor Ippolitov
     +   Copyright 2023-2024 Taryn Musgrave
         Copyright 2021-2023 Alejandro Colomar
     -   Copyright 2022-2023 Andrew Clayton
    @@ pkg/rpm/rpmbuild/SOURCES/COPYRIGHT.unit-jsc8
         Copyright 2017-2022 Max Romanov
         Copyright 2021-2022 Oisín Canty
     
    + ## src/test/nxt_rbtree1.c ##
    +@@
    + 
    + /*
    +  * Copyright (C) Igor Sysoev
    +- * Copyright (C) Nginx, Inc.
    ++ * Copyright (C) NGINX, Inc.
    +  */
    + 
    + 
    +
    + ## src/test/nxt_rbtree1.h ##
    +@@
    + 
    + /*
    +  * Copyright (C) Igor Sysoev
    +- * Copyright (C) Nginx, Inc.
    ++ * Copyright (C) NGINX, Inc.
    +  */
    + 
    + 
    +
      ## tools/setup-unit ##
     @@ tools/setup-unit: unit_ctl_welcome()
      
```

File: docs/man/man8/unitd.8.in [hunk 2]
```diff
@@ -1,10 +1,16 @@
-.\" (C) 2017-2023, NGINX, Inc.
-.\" (C) 2017-2023 Andrei Zeliankou
-.\" (C) 2018-2023 Konstantin Pavlov
-.\" (C) 2021-2023 Zhidao Hong
+.\" (C) 2017-2024, NGINX, Inc.
```
Comment (javorszky) [inline docs/man/man8/unitd.8.in:1]: super nitpick, feel free to ignore, not a blocker, why does this line have a , after the years, but not the lines with the individuals' names? 🙂

**Comment by javorszky** [comment]
The order from what I can tell is:
* current contribs first, past contributors later
* sorted by oldest "from" year
* within the same year range alphabetical?
* where NGINX, Inc is there, that comes at the top of the list

Is that accurate? 🙂

File: docs/man/man8/unitd.8.in (hunk 2 above)
Comment (callahad) [reply docs/man/man8/unitd.8.in:1]: Occurs again below...

Comment (callahad) [reply docs/man/man8/unitd.8.in:1]: LGTM minus fixing the spurious initial commas in "2024, NGINX, Inc."

Comment (andrey-zelenkov) [reply docs/man/man8/unitd.8.in:1]: No idea, will remove this one.

**Comment by andrey-zelenkov** [comment]
Rebase and refine:

```
 1:  11b90144 ! 42:  fb7eead2 Updated copyright notice.
    @@ docs/man/man8/unitd.8.in
     -.\" (C) 2017-2023 Andrei Zeliankou
     -.\" (C) 2018-2023 Konstantin Pavlov
     -.\" (C) 2021-2023 Zhidao Hong
    -+.\" (C) 2017-2024, NGINX, Inc.
    ++.\" (C) 2017-2024 NGINX, Inc.
     +.\" (C) 2017-2024 Andrei Zeliankou
     +.\" (C) 2018-2024 Konstantin Pavlov
     +.\" (C) 2021-2024 Zhidao Hong
    @@ docs/man/man8/unitd.8.in: The socket address of Unit's control API.
     -(C) 2017-2023 Andrei Zeliankou
     -(C) 2018-2023 Konstantin Pavlov
     -(C) 2021-2023 Zhidao Hong
    -+(C) 2017-2024, NGINX, Inc.
    ++(C) 2017-2024 NGINX, Inc.
     +(C) 2017-2024 Andrei Zeliankou
     +(C) 2018-2024 Konstantin Pavlov
     +(C) 2021-2024 Zhidao Hong
```

**Review by callahad — APPROVED** [review]

**Pull request merged**